SilKit/IntegrationTests/ITest_MessageAggregation.cpp

```diff
@@ -105,8 +105,8 @@ TEST_F(ITest_MessageAggregation, timeout_in_case_of_deadlock_when_using_async_si
         });
 
         timeSyncService->SetSimulationStepHandlerAsync(
-            [dataPublisher, lifecycleService, timeSyncService, &msgReceived](std::chrono::nanoseconds /*now*/,
-                                                                             std::chrono::nanoseconds /*duration*/) {
+            [dataPublisher, lifecycleService, &msgReceived](std::chrono::nanoseconds /*now*/,
+                                                                  std::chrono::nanoseconds /*duration*/) {
```

Comment:
It seems that formatting is broken in this line.

Reply:
Good catch! Thank you 👍